DDDEastAnglia/Helpers/LoginMethods/LoginMethodIconProvider.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using DDDEastAnglia.Helpers.AppSettings;
+
+namespace DDDEastAnglia.Helpers.LoginMethods
+{
+    public class LoginMethodIconProvider
```

Comment:
Could this become a `LoginMethodIcon`? Or perhaps it should just be a property of a `LoginMethod` in general.